### ISSUE TITLE
[Feature]: Selective muting in groups

### ISSUE THREAD

**Issue by MoonlitMonitor** [simplex-chat/simplex-chat#2881]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### Feature

There are cases in which you may want to mute notifications from a specific user in a group, but not from other users in that group. 

For example, if you are in a group chat with another person, but also with another device of yours so that you can access the group chat from multiple devices, then there is no reason to be notified across your devices when you receive a message that you sent from another device.

In other words, if there are three participants in a group chat: Bob's Phone, Bob's PC, and Alice, then there is no reason for Bob's PC to be notified when Bob's Phone sends a message, and there is no reason for Bob's Phone to be notified when Bob's PC sends a message. Bob's PC and Bob's Phone just want to receive notifications from Alice, while Alice wants to receive notifications from Bob's PC and Bob's Phone.

Therefore it would be useful if you could selectively mute users within a group.
